src/App.tsx

```diff
@@ -1,223 +1,170 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext, useEffect, useRef, useState,
+} from 'react';
 import { AuthContext } from './components/Auth/AuthContext';
+import { ErrorNotification } from './components/Error/ErrorNotification';
+import { TodoList } from './components/TodoList/TodoList';
+import { getTodos, addTodo, deleteTodo } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoCount } from './components/TodoCount/TodoCount';
```

Comment:
if you add `index.ts` in each folder with your components and write in this file 
`export * from './<fileWithExportedComponent>.tsx';`
your imports can be shorter
```suggestion
import { TodoCount } from './components/TodoCount';
```